src/Pretzel/Program.cs

```diff
@@ -102,6 +103,7 @@ private void LoadPlugins(string[] commandArgs)
                 if (Directory.Exists(pluginsPath))
                 {
                     catalog.Catalogs.Add(new DirectoryCatalog(pluginsPath));
+                    catalog.Catalogs.Add(new ScriptCsCatalog(pluginsPath, new ScriptCsCatalogOptions { References = new[] { typeof(DotLiquid.Tag), typeof(Pretzel.Logic.Extensibility.ITag) } }));
```

Comment:
Instead of hardcoding the dependency on scriptcs, you could find the catalog and dynamically load it. There's a host of diff strategies you could use to do this. One could be checking if the assembly exists, and if it does you dynamically load it and instantiate yourself via reflection. You could move `ScriptcsCatalogOptions` to a separate `Pretzel.ScriptCs.Contracts` package so that Pretzel and the Pretzel.ScriptCs package can reference it, this way Pretzel can still construct and pass in the options easily.